src/cmake/linux.cmake

```diff
@@ -112,6 +112,8 @@ install(FILES platforms/linux/daemon/org.mozilla.vpn.conf
 install(FILES platforms/linux/daemon/org.mozilla.vpn.dbus.service
     DESTINATION ${CMAKE_INSTALL_DATADIR}/dbus-1/system-services)
 
+add_definitions(-DADDONS_PATH=\"${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_DATADIR}/mozillavpn/addons\")
```

Comment:
We should really be moving away from using hard-coded paths like this, and we should instead perform a runtime search using the `XDG_DATA_DIRS` environment variable.

Reply:
I removed this part.